src/universal/modules/email/components/Card/Card.js

```diff
@@ -53,6 +55,9 @@ const Card = (props) => {
     };
   }
 
+  const markdownCustomComponents = {
```

Comment:
let's move this outside the component since it'll never change & then we can reuse that same object for all the cards.